src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/base_js.html.twig

```diff
@@ -171,6 +175,23 @@
 
         var finishAjaxRequest = function(index) {
             var request = requestStack[index];
+
+            if (request.toolbarReplace && !request.toolbarReplaceFinished && request.profile) {
+                /* flag as complete because finishAjaxRequest can be called multiple times */
+                request.toolbarReplaceFinished = true;
+                Sfjs.load(
+                    Sfjs.toolbar.id,
+                    Sfjs.toolbar.url(request.profile),
+                    Sfjs.toolbar.success(request.profile),
+                    function(xhr) {
```

Comment:
`Sfjs.toolbar.error()`? This is needed twice basically.

Reply:
I think we can even wrap `Sfjs.load()` entirely, i.e. `Sfjs.doLoad(token)`.

Reply:
I suggest creating a `Sfjs.loadToolbar(token)` method (and stop exposing `Sfjs.toolbar.*`)